GpxExport/ViewControllers/WorkoutsTableViewController.swift

```diff
@@ -90,6 +90,19 @@ class WorkoutsTableViewController: UITableViewController {
     }
 
     private func exportSelectedWorkouts() {
+        let alert = UIAlertController(title: "What file to generate?", message: "It's recommended to use GPX for now.", preferredStyle: .actionSheet)
```

Comment:
The strings should be localized.

Reply:
Yeah, of course. I forgot that we do that now ;)